### PR TITLE
Added SimpleDraweeView.setImageURI(@Nullable String) method

### PR DESCRIPTION
**Pull request by froxCZ**
also changed one example in demo app to show the new method

### CODE REVIEW

File: drawee/src/main/java/com/facebook/drawee/view/SimpleDraweeView.java [hunk 1]
```diff
@@ -111,4 +120,18 @@ public void setImageURI(Uri uri, @Nullable Object callerContext) {
...
   }
+
+  /**
+   * Displays an image given by the uri string.
+   *
+   * @param uriString uri string of the image
+   * @param callerContext caller context
+   */
+  public void setImageURI(@Nullable String uriString, @Nullable Object callerContext) {
+    Uri uri = null;
```
Comment (plamenko) [inline drawee/src/main/java/com/facebook/drawee/view/SimpleDraweeView.java]: Uri uri = (uriString != null) ? Uri.parse(uriString) : null;

**Comment by plamenko** [comment]
Looks good, can you please address the tiny comment?

**Comment by froxCZ** [comment]
@plamenko any idea when is this going to be merged? Thanks!

**Comment by oprisnik** [comment]
@froxCZ We'll merge it soon.

@facebook-github-bot import

**Pull request closed without merging**